src/App.js

```diff
@@ -3,15 +3,20 @@ import {BrowserRouter as Router} from 'react-router-dom';
 import Routes from './Routes';
 import {ApolloProvider} from 'react-apollo';
 import {client} from './state/client';
+import {AnalyticsTrackingProvider} from './analyticsTracking';
 
-const App = () => (
-  <ApolloProvider client={client}>
-    <Router>
-      <main className="App">
-        <Routes />
-      </main>
-    </Router>
-  </ApolloProvider>
-);
+const App = () => {
+  return (
+    <AnalyticsTrackingProvider>
```

Comment:
I think you can get rid of the custom provider here and just use amplitude:
```
const App = () ={
  const [user] = useQuery(MY_PROFILE);
  return (
    <Amplitude
        eventProperties={inheritedProps => ({
          ...inheritedProps,
          user: user
        })}
    >
      <Router />
    </Amplitude>
  )
}
```

Reply:
Amplitude still requires an AmplitudeProvider at the top

Reply:
Yeah you still have that on line L11.
I think you can ditch the `AnalyticsTrackingProvider` and `AmplitudeUser` for the above code.

Reply:
AnalyticsTrackingProvider now user AmplitudeProxy